xbmc/pictures/GUIWindowPictures.cpp

```diff
@@ -502,6 +502,8 @@ void CGUIWindowPictures::GetContextButtons(int itemNumber, CContextButtons &butt
   CGUIMediaWindow::GetContextButtons(itemNumber, buttons);
   if (item && !item->GetProperty("pluginreplacecontextitems").asBoolean())
     buttons.Add(CONTEXT_BUTTON_SETTINGS, 5);                  // Settings
+
+  BaseContextMenuManager::Get().AppendVisibleContextItems(item, buttons);
```

Comment:
Would be nice if we didn't have to manually add this to every `GetContextButtons()` implementation we have.